roles/keystone/tasks/federation.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: install mod_auth_openidc deps
+  apt: name={{ item }} state=present
+  with_items:
+    - libjansson4
+    - libhiredis0.10
+    - libcurl3
+
+- name: download mod_auth_openidc
+  get_url: url={{ keystone.federation.sp.oidc.download.url }}
+           dest=/tmp/libapache2-mod-auth-openidc_{{ keystone.federation.sp.oidc.version }}-1_amd64.deb
+
+- name: install mod_auth_openidc
+  apt: deb=/tmp/libapache2-mod-auth-openidc_{{ keystone.federation.sp.oidc.version }}-1_amd64.deb
```

Comment:
Pulled this off of what the site controller does. Still nice to have the version in there so that if we move to a newer version we can change it in one place. I will update how we do this.